dubbo-common/src/main/java/org/apache/dubbo/common/utils/UrlUtils.java

```diff
@@ -445,6 +451,22 @@ public static boolean isServiceKeyMatch(URL pattern, URL value) {
         return urls.stream().filter(predicate).collect(Collectors.toList());
     }
 
+    public static boolean isConfigure(URL url) {
+        return CONFIGURATORS_CATEGORY.equals(url.getParameter(Constants.CATEGORY_KEY, Constants.DEFAULT_CATEGORY))

```

Comment:
I think below one would more better from symmetry to other and better from performance point of view (although negligible 😄 )

return OVERRIDE_PROTOCOL.equals(url.getProtocol()) ||   CONFIGURATORS_CATEGORY.equals(url.getParameter(Constants.CATEGORY_KEY, Constants.DEFAULT_CATEGORY));